src/vmm/src/arch/aarch64/regs.rs

```diff
@@ -214,15 +208,17 @@ pub fn setup_boot_regs(
     // Get the register index of the PSTATE (Processor State) register.
     let pstate = offset__of!(user_pt_regs, pstate) + kreg_off;
     let id = arm64_core_reg_id!(KVM_REG_SIZE_U64, pstate);
-    vcpu.set_one_reg(id, PSTATE_FAULT_BITS_64.into())
+    vcpufd
+        .set_one_reg(id, PSTATE_FAULT_BITS_64.into())
         .map_err(|err| Error::SetOneReg(id, err))?;
 
-    // Other vCPUs are powered off initially awaiting PSCI wakeup.
+    // Other vcpufds are powered off initially awaiting PSCI wakeup.
```

Comment:
Nit: I guess it's vCPUs that are powered off, not vcpufds?

Reply:
fixed